README.md

```diff
@@ -757,6 +758,22 @@ const systemVersion = DeviceInfo.getSystemVersion();
 
 ---
 
+### getBuildId()
+
+Gets build number of the operating system.
+
+**Examples**
+
+```js
+const osBuildId = DeviceInfo.getBuildId();
+
+// iOS: "12A269"
+// Android: "13D15"
+// Windows: not support
+```
```

Comment:
Since "not available" is a valid return value for the function could you add that in the example? so people will know to test for it...

Reply:
Yeah, I should change the documentation for valid return for Windows.
Done 👍